integration-tests/integration/caseNotes/case-note-amendments.cy.js

```diff
@@ -94,7 +96,8 @@ context('Case note amendments', () => {
     page.moreDetail().type('Hello, world!')
     page.save().click()
 
-    CaseNotesPage.verifyOnPage('Smith, John')
+    // Weirdly a user can create the case note but can't access the case note page...
```

Comment:
Uncovering some interesting facts about how users with no caseloads would have been able to interact with the old profile!